src/js/components/AppPageComponent.jsx

```diff
@@ -343,11 +343,8 @@ var AppPageComponent = React.createClass({
     var volumes = AppsStore.getVolumes(model.id);
 
     var tabs = state.tabs.filter(tab =>
-      tab.text !== "Volumes" || (
-        model.container != null &&
-        model.container.volumes != null &&
-        model.container.volumes.length > 0
-      ));
```

Comment:
@Poltergeist off the top of your head, what was the reason for inspecting `model.container.volumes` vs the `volumes` array 2 lines above as I've done in my PR? The `getVolumes` function checks for the presence of tasks. 
Do you agree that we don't want to show the Volumes tab unless at least a task is running? Have I missed some other piece of logic here?